apis/Google.Cloud.Diagnostics.AspNet/docs/index.md

```diff
@@ -9,6 +9,11 @@ It allows for simple integration of Stackdriver into ASP.NET applications with m
 
 {{auth}}
 
+# Note
+The Google.Cloud.Diagnostics.AspNet package attempts to collect the filename and line number when
+some entries are collected.  However, to be able to collect this information PDBs must be included with
```

Comment:
Not sure what "some entries" means - is there any particular criterion for this? Or is it all entries? (Maybe just remove "some"?)

Fine other than that.

Reply:
no particular criterion, updated. Thanks!